src/ports/postgres/modules/deep_learning/madlib_keras_automl.py_in

```diff
@@ -17,27 +17,41 @@
 # specific language governing permissions and limitations
 # under the License.
 
+from ast import literal_eval
 from datetime import datetime
-import plpy
+from hyperopt import hp, rand, tpe, atpe, Trials, STATUS_OK, STATUS_RUNNING
+from hyperopt.base import Domain
 import math
+import numpy as np
+import plpy
 from time import time
 
 from madlib_keras_validator import MstLoaderInputValidator
-from utilities.utilities import unique_string, add_postfix, extract_keyvalue_params, \
-    _assert, _assert_equal, rename_table
+from utilities.utilities import get_seg_number, get_segments_per_host, unique_string, add_postfix, \
+    extract_keyvalue_params, _assert, _assert_equal, rename_table
 from utilities.control import MinWarning, SetGUC
 from madlib_keras_fit_multiple_model import FitMultipleModel
 from madlib_keras_model_selection import MstSearch, ModelSelectionSchema
 from keras_model_arch_table import ModelArchSchema
 from utilities.validate_args import table_exists, drop_tables
 
+import inspect
+
+def decallmethods(decorator, prefix='test_'):
```

Comment:
We should use `_` in the function name. I wasn't sure if this was meant to be `de-call methods` or `dec all methods`. Also a short explanation on what this function and the inner class doing would be great.

Reply:
I inserted this to make the class inheritance work earlier. On further inspection, it seems I may not need this function.